src/core/Database.cpp

```diff
@@ -118,6 +123,56 @@ Entry* Database::recFindEntry(const Uuid& uuid, Group* group)
     return nullptr;
 }
 
+Entry *Database::recFindEntry(const QString &text, EntryReferenceType referenceType, Group *group)
```

Comment:
I don't really like this function's name. But the other one is named like this as well, so nevermind

Reply:
In my opinion it would be better to use a name like the following **findEntryRecursive** and **findGroupRecursive**. What do you think about this?

Reply:
I like them more :+1: